code/src/App.js

```diff
@@ -1,9 +1,81 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Summary } from './Summary'
+import { Header } from './Header'
+
+const sodaFavorite = ["fanta", "pepsi", "sprite"];
 
 export const App = () => {
+
+  const [pizza, setPizza] = useState("");
+  const [soda, setSoda] = useState();
+  const [sauce, setSauce] = useState();
+  const [submitted, setSubmitted] = useState(false)
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <section> 
+
+      <Header />
+
+        {!submitted && (
+
+        <form onSubmit={event => event.preventDefault()}>
+
+          <section className="firstSection">
+
+          <p>Which type of pizza you prefer?</p>
+          <select className="formDropdown"
+            onChange={event => setPizza(event.target.value)}
+            value={pizza}
+          >
+            <option value="">Choose your favorite!</option>
```

Comment:
I guess this isn't a option
```suggestion
            <option value="" disabled >Choose your favorite!</option>
```